inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -298,8 +298,8 @@ CREATE TABLE IF NOT EXISTS `inlong_stream_field`
     `field_type`          varchar(20)  NOT NULL COMMENT 'field type',
     `field_comment`       varchar(50)  DEFAULT NULL COMMENT 'Field description',
     `is_meta_field`       smallint(3)  DEFAULT '0' COMMENT 'Is this field a meta field? 0: no, 1: yes',
-    `meta_field_name`     varchar(20)  DEFAULT NULL COMMENT 'Meta field name',
-    `field_format`        varchar(50)  DEFAULT NULL COMMENT 'Field format, including: MICROSECONDS, MILLISECONDS, SECONDS, SQL, ISO_8601 and custom such as yyyy-MM-dd HH:mm:ss',
+    `meta_field_name`     varchar(120)  DEFAULT NULL COMMENT 'Meta field name',
+    `field_format`        text  DEFAULT NULL COMMENT 'Field format, including: MICROSECONDS, MILLISECONDS, SECONDS, SQL, ISO_8601 and custom such as yyyy-MM-dd HH:mm:ss',
```

Comment:
Why does `field_format` need text type? In what scenario does it store long strings?

Reply:
> Why does `field_format` need text type? In what scenario does it store long strings?

`field_format` can store serialize format of complex type. It cloud be too long. So I set text.

Reply:
Okay, please add this scenario to the comment info, thanks.

Reply:
done